docs/databases/swift-kuery.html

```diff
@@ -132,20 +141,21 @@ <h3 class="heading-3">Linux</h3>
           <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create a connection to an SQLite database</h2>
           <p>First we need to import the SwiftKuery and SwiftKuerySQLite packages.</p>
           <pre><code class="language-swift">import SwiftKuery
-  import SwiftKuerySQLite</code></pre>
-          <p>Next we create a connection pool of connections:</p>
-          <pre><code class="language-swift">let pool = SQLiteConnection.createPool(filename: "myDB.db", poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))</code></pre>
+import SwiftKuerySQLite</code></pre>
+          <p>Inside the `App` extension, create a connection pool of connections:</p>
+          <pre><code class="language-swift">let pool = SQLiteConnection.createPool(filename: "myDB.db", poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))
```

Comment:
We don't have a step in either MySQL or SQLite to create a database, so PostgreSQL has steps 2a, 2b and 2c whereas the other two don't, is this just an oversight?